pset-iterators.js

```diff
@@ -93,6 +115,17 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+const presidentialParties = []
+
+// Filter the Presidents and extract their respective
+// Party to the presidentialParties array
+const cyclePresidents = presidents.filter(function(president) {
```

Comment:
@ksuitor technically this works, but the more effective approach was to use `.map()` which iterates through the array and generates a new array based on the logic placed in the callback, see the solution for more details: https://github.com/jsd20191008/hw-02-iterators-solution/blob/master/pset-iterators.js#L98